### PR TITLE
update gitignore, and include migrations

### PR DESCRIPTION
**Pull request by otakup0pe**
The docker entrypoint script never made it in with #24 because `.gitignore` is a cop. This PR updates the gitignore and includes the script. It also includes the migrations which were previously ignored.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -213,721 +167,29 @@ cython_debug/
...
 node_modules
+bower_components
 
 *.sublime*
 
 psd
 thumb
 sketch
 
-### Vim ###
```
Comment (Index01) [inline .gitignore:507]: Def need the vim stuffs. Else my world goes sideways.
I'll try to pull and add to this MR tonight then merge

Comment (Index01) [reply .gitignore:507]: Lulz, tho I now see that is the section that was giving trouble with [Ss]cripts. 😆 I'll trim it down to just the bits I needs

Comment (otakup0pe) [reply .gitignore]: Good call on the editor bits. Added stuff for vim and emacs.

Comment (Index01) [reply .gitignore]: Lgtm mergying!

**Pull request merged**